### DISCUSSION TITLE
Uninstallers location

### DISCUSSION THREAD

**Discussion by XXLMandalorian013** [microsoft/winget-pkgs#92100] [General] [answered]
Hello,

I was not seeing where the uninstallers were located. I was going to create a pull request for DropBox's uninstaller. Could someone please point me in the right direction?

**Comment by Trenly** [comment, marked as answer]
> Hello,
> 
> I was not seeing where the uninstallers were located. I was going to create a pull request for DropBox's uninstaller. Could someone please point me in the right direction?

Uninstallers don't need manifests, only installers do. When running an uninstall, winget will use whichever uninstaller the installer registered to windows when the application was first installed, regardless of if the application was installed with winget or not

**Comment by XXLMandalorian013** [comment]
Ah, thanks!

**Comment by teke85** [comment]
Why am I receiving this
